src/components/pages/dashboard/ManageInstance/cmp.tsx

```diff
@@ -221,8 +221,7 @@ export default function ManageInstance() {
 
                         <Link
                           className="tp-body1 fs-16"
-                          href={'?hash=' + key.id}
-                          referrerPolicy="no-referrer"
```

Comment:
Why are we removing the referrerPolicy value here?

Reply:
Other similar links were not using it in other parts of the code, so for the consistency. I let you judge